app/upgrades/v15/upgrades.go

```diff
@@ -64,33 +121,201 @@ func UpgradeSigningInfos(ctx sdk.Context, sk slashingkeeper.Keeper) {
 	}
 
 	ctx.Logger().Info("Finished migrating signing infos")
+	return nil
 }
 
-// UpgradeMinCommissionRate sets the minimum commission rate staking parameter to 5%
-// and updates the commission rate for all validators that have a commission rate less than 5%
-// adhere to prop 826 which sets the minimum commission rate to 5% for all validators
-// https://www.mintscan.io/cosmos/proposals/826
-func UpgradeMinCommissionRate(ctx sdk.Context, sk stakingkeeper.Keeper) {
-	ctx.Logger().Info("Migrating min commission rate...")
+// ClawbackVestingFunds transfers the vesting tokens from the given vesting account
+// to the community pool
+func ClawbackVestingFunds(ctx sdk.Context, address sdk.AccAddress, keepers *keepers.AppKeepers) error {
+	ctx.Logger().Info("Migrating vesting funds...")
 
-	params := sk.GetParams(ctx)
-	params.MinCommissionRate = sdk.NewDecWithPrec(5, 2)
-	err := sk.SetParams(ctx, params)
-	if err != nil {
-		panic(err)
+	ak := keepers.AccountKeeper
+	bk := keepers.BankKeeper
+	dk := keepers.DistrKeeper
+	sk := *keepers.StakingKeeper
+
+	// get target account
+	account := ak.GetAccount(ctx, address)
+
+	// verify that it's a vesting account type
+	vestAccount, ok := account.(*vesting.ContinuousVestingAccount)
+	if !ok {
```

Comment:
Not returning an error here shouldn't affect the states and prevent breaking the upgrade test of the git workflow.